assets/js/admin/upgrades/v3/index.js

```diff
@@ -0,0 +1,182 @@
+const EDD_v3_Upgrades = {
```

Comment:
It doesn't make a difference in the end, but this object pattern is not needed with Webpack. Everything is taken out of the global namespace automatically with the module imports so you can just use normal functions if you want.

Reply:
Thanks for letting me know! We'll see if I can be bothered to refactor it. :D